Caboodle/PhoneDialer/PhoneDialer.ios.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Foundation;
+using UIKit;
+using CoreTelephony;
+
+namespace Microsoft.Caboodle
+{
+	public static partial class PhoneDialer
+	{
+		public static bool IsSupported
+		{
+			get
+			{
+				var nsUrl = CreateNsUrl("0000000000");
+				var canCall = UIApplication.SharedApplication.CanOpenUrl(nsUrl);
+				
+				if (canCall)
+				{
+					using (var netInfo = new CTTelephonyNetworkInfo())
```

Comment:
What all is this check doing?

Reply:
we are checking for SIM-card & network availability